src/Router.tsx

```diff
@@ -4,6 +4,7 @@ import Home from "./components/Home";
 import Example from "./components/Example";
 import Counter from "./components/Counter";
 import Testing from "./components/Testing";
+import Footer from "./components/footer/Footer";
```

Comment:
name the component `Footer.js` to `index.js` in the `footer` folder. That way the import will be simpler `"./components/footer"`